config-linux.md

```diff
@@ -460,25 +460,34 @@ The following parameters can be specified to set up the controller:
 **`intelRdt`** (object, OPTIONAL) represents the [Intel Resource Director Technology][intel-rdt-cat-kernel-interface].
     If `intelRdt` is set, the runtime MUST write the container process ID to the `<container-id>/tasks` file in a mounted `resctrl` pseudo-filesystem, using the container ID from [`start`](runtime.md#start) and creating the `<container-id>` directory if necessary.
     If no mounted `resctrl` pseudo-filesystem is available in the [runtime mount namespace](glossary.md#runtime-namespace), the runtime MUST [generate an error](runtime.md#errors).
-
     If `intelRdt` is not set, the runtime MUST NOT manipulate any `resctrl` pseudo-filesystems.
 
 The following parameters can be specified for the container:
 
 * **`l3CacheSchema`** *(string, OPTIONAL)* - specifies the schema for L3 cache id and capacity bitmask (CBM).
-    If `l3CacheSchema` is set, runtimes MUST write the value to the `schemata` file in the `<container-id>` directory discussed in `intelRdt`.
+    The value SHOULD start with `L3:` and SHOULD NOT contain newlines.
+* **`memBwSchema`** *(string, OPTIONAL)* - specifies the schema of memory bandwidth percentage per L3 cache id.
+    The value MUST start with `MB:` and MUST NOT contain newlines.
+
+    If both `l3CacheSchema` and `memBwSchema` are set, runtimes MUST write the combined value to the `schemata` file in the <container-id> directory discussed in intelRdt. If `l3CacheSchema` contains a line beginning with `MB:`, the value written to `schemata` file MUST be the non-`MB:` line(s) from `l3CacheSchema` and the line from `memBWSchema`.
```

Comment:
I'd drop the indent here to make it a paragraph following the list (it's currently a paragraph inside the `memBwSchema` list entry).  This (and the following paragraphs) are about both `l3CacheSchema` and `memBwSchema`, and the `memBwSchema` list entry seems like too specific a location for that information.

You're also dropping the previous backticks around `<container-id>`, and we want to keep those because Markdown is a subset of HTML, so an unticked version will be invisible (as an unrecognized HTML tag).

And you have two sentences in one line, when [prefer one sentence per line][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/style.md#one-sentence-per-line

Reply:
@wking 
Thank you. I will fix these issues.